### PR TITLE
Stealth Generator Power

### PR DESCRIPTION
**Pull request by CDVoidwalker**

### CODE REVIEW

**Review by jrb0001 — CHANGES_REQUESTED** [review]
The conditions `cloakgenerator`, `crate-cloak` and `cloakgeneratorpower` should be merged into `cloaked` and `cloaked-bar`.

**Comment by GraionDilach** [comment]
Disagree with the change requested. It's better to keep them separate for easier override on a per-unit basis like on the Stealth Tank (which shouldn't be able to pick up cloak crates IMO).

**Comment by CDVoidwalker** [comment]
Well, our crates are all random so I guess it won't have much value to keep those conditions separated

File: rules/defaults.yaml [hunk 1]
```diff
@@ -326,21 +326,19 @@
...
-		Condition: crate-cloak
-	ExternalCondition@EXTERNALCLOAK3:
-		Condition: cloakgeneratorpower
-	#TimedConditionBar@cloakgeneratorpower: #change it if you want to make it non perma
-		#Condition: cloakgeneratorpower
+	ExternalCondition@EXTERNALCLOAK:
+		Condition: cloak
+	ExternalCondition@EXTERNALCLOAKBAR:
+		Condition: cloak-bar
+	#TimedConditionBar@cloakedbar: #change it if you want to make it non perma
```
Comment (jrb0001) [inline rules/defaults.yaml]: This seems to be redundant?

File: rules/forgotten-infantry.yaml [hunk 2]
```diff
@@ -167,7 +167,7 @@ EXPLORER:
 	AttackFrontal:
 	-ProducibleWithLevel:
 	Cloak:
-		Condition: crate-cloak || cloakgenerator
+		Condition: cloak || cloakgenerator
```
Comment (jrb0001) [inline rules/forgotten-infantry.yaml]: Should be `cloak || cloak-bar`.

**Review by jrb0001 — CHANGES_REQUESTED** [review]

**Review by jrb0001 — APPROVED** [review]

**Pull request merged**